src/github-repo.ts

```diff
@@ -220,11 +220,15 @@ export class GithubRepo {
       (a) => a.name === assetName
     );
 
+    const existingAssets = releaseDetails.assets?.map((a) => a.name);
+
     if (existingAsset) {
       await this.octokit.repos.deleteReleaseAsset({
         ...this.repo,
         asset_id: existingAsset.id,
       });
+    } else {
+      console.log('could not find the asset in the existing assets', { assetName, existingAssets });
```

Comment:
i think this package also uses `debug` somehow

Reply:
It doesn't. I can add it, but then we have to make sure the debug output isn't filtered out.. somewhere.

Reply:
Also debug doesn't have an equivalent for console.dir() as far as I know :)